app/src/atoms/buttons/LargeButton.tsx

```diff
@@ -114,29 +138,40 @@ export function LargeButton(props: LargeButtonProps): JSX.Element {
     box-shadow: none;
     padding: ${SPACING.spacing24};
     line-height: ${TYPOGRAPHY.lineHeight20};
-    border: ${LARGE_BUTTON_PROPS_BY_TYPE[buttonType].border};
+    border: ${BORDERS.borderRadius4} solid
```

Comment:
looks like the gap between the text and the icon has been updated.

<img width="303" alt="Screenshot 2024-06-25 at 3 14 19 PM" src="https://github.com/Opentrons/opentrons/assets/474225/20483512-d681-4c32-940b-8046a40ff27c">


```suggestion
    grid-gap: ${SPACING.spacing60}
    border: ${BORDERS.borderRadius4} solid
```

Reply:
Oh, good catch